server_test.go

```diff
@@ -167,6 +198,153 @@ func TestLocalServer(t *testing.T) {
 	server.Flush(config.Interval, config.FlushLimit)
 }
 
+func TestLocalServerMixedMetrics(t *testing.T) {
+
+	// Same as in TestLocalServerUnaggregatedMetrics
+	var RemoteResponses int
+	defer func() {
+		assert.Equal(t, 1, RemoteResponses, "Server did not complete all responses before test terminated!")
+	}()
+
+	// Unlike flushing to to the remote server, flushing forward
+	// (ie, flushing to global veneur) is asynchronous.
+	// This means it is quite likely that the function will return before
+	// the global server has actually responded, unless we introduce a small timeout
+	GlobalResponseChan := make(chan struct{})
+	defer func() {
+		select {
+		case <-GlobalResponseChan:
+			// all is safe
+		case <-time.After(DefaultServerTimeout):
+			assert.Fail(t, "Global server did not complete all responses before test terminated!")
+		}
+	}()
+
+	// We can't be guaranteed that we will receive this exact stream
+	// but we should expect that they represent the same underlying tdigest
+	const ExpectedGobStream = "\r\xff\x87\x02\x01\x02\xff\x88\x00\x01\xff\x84\x00\x007\xff\x83\x03\x01\x01\bCentroid\x01\xff\x84\x00\x01\x03\x01\x04Mean\x01\b\x00\x01\x06Weight\x01\b\x00\x01\aSamples\x01\xff\x86\x00\x00\x00\x17\xff\x85\x02\x01\x01\t[]float64\x01\xff\x86\x00\x01\b\x00\x00/\xff\x88\x00\x05\x01\xfe\xf0?\x01\xfe\xf0?\x00\x01@\x01\xfe\xf0?\x00\x01\xfe\x1c@\x01\xfe\xf0?\x00\x01\xfe @\x01\xfe\xf0?\x00\x01\xfeY@\x01\xfe\xf0?\x00\x05\b\x00\xfeY@\x05\b\x00\xfe\xf0?\x05\b\x00\xfeY@"
+
+	var HistogramValues = []float64{1.0, 2.0, 7.0, 8.0, 100.0}
+	const ABCCountRaw = 5
+	const ABCCountNormalized = 100
```

Comment:
nit: magic number. let's define a constant for the server's flush interval (ie 50ms) and then, instead of hardcoding this, we can compute it as `ABCCountRaw * time.Second / DefaultFlushInterval`